### PR TITLE
fix #271 - give feedback for unrecognized commands

### LINKED ISSUE #271: CLI is silent when passed an unrecognized option
**Describe the bug**
The `lingui` CLI is silent when it is passed an unrecognized option.

**To Reproduce**
`$ lingui foo`

**Expected behavior**
The CLI should display an error.

`lingui --version`: 2.4.1

### PR DESCRIPTION
**Pull request by dephiros**
fix #271

### CODE REVIEW

File: packages/cli/src/lingui.js [hunk 1]
```diff
@@ -19,3 +21,5 @@ program
   .command("extract [files...]", "Extracts messages from source files")
   .command("compile", "Compile message catalogs")
   .parse(process.argv)
+
+console.log(fuzzValidateCommand(program.commands, program.args))
```
Comment (FredyC) [inline packages/cli/src/lingui.js]: Wouldn't it print an empty line for valid cases? Might be better to check for the result before printing it.

Comment (dephiros) [reply packages/cli/src/lingui.js]: Thank you. I will add that check

**Comment by dephiros** [comment]
Realized that I forgot to push up the test file. Will push it up tonight

**Comment by tricoder42** [comment]
@dephiros Yeah, don't worry about it. I honestly don't know why the test fails, but I rerun the test suit and everything is fine.

I'm gonna merge it in the morning. Great job, thank you very much! ⚡️ ⚡️ ⚡️

**Pull request merged**